products/ward round documentation/new feature description_ward round documentation.md

```diff
@@ -0,0 +1,248 @@
+    - Etc.
+- Particularly relevant for ICUs:
+    - Vital signs (in case of ICU dynamically displayed, otherwise collected from manual entries)
+    - List of currently used equipment (e.g. intravenous catheters, arterial indwelling catheters, central lines, drainages, bladder catheter), each with date of insertion and automated reminder to change the material, e.g. as a helpwave task (intervals can previously be determined based on the local SOPs = standard operating procedures)
+- Appointments
+    - List of important planned appointments, e.g.:
+    - Therapy (surgery, dialysis, etc.)
+    - Diagnostics (beyond the scope of the own ward, e.g. imaging)
+    - External appointments (rehabilitation, ambulatory follow-up, etc.)
+    - Status of planned discharge or transferal to a different ward or medical facility
+    - Possibility to register new appointments directly (e.g. chest x-ray) or communicate with the patient management team (without having to write an e-mail)
+    - Possibility to view ward occupancy plan, ward calendar, etc.
+- Scores and scales
+    - “one-click” documentation of relevant scores and scales (e.g. NRS, SOFA, GCS, RASS)
+    - Image documentation (e.g. for wound management)
+
+___
+
+## 4. User Stories
+
+    - List several user stories or scenarios, which are brief descriptions of how different user roles (doctors, nurses, pharmacists, administrative staff etc.) will interact with the feature.
+    - Each user story should follow the format: "As a [user role], I want to [action] so that [benefit]." (e.g., "As a doctor, I want to quickly access a patients' medication history so that I can make informed prescribing decisions.")
+
+- What types of ward rounds are there?
+    1. Doctors daily round on each patient in their ward. Ward Rounds usually include all junior doctors that are assigned to the ward as well as some senior doctors. It is common that nurses join the ward round only for those patients that they are responsible for, so that they can give additional information if needed.
+    2. Nurses usually round on patients at the beginning of their shift to pass on relevant information between from one shift to the next. The relevant information differs from the doctor ward rounds. Usually there are no decisions to be made regarding diagnostics and therapy.
+    3. Interdisciplinary ward rounds take place in settings such as ICU (intensive care unit) and IMC (intermediate care unit). This might include the following employee groups: doctors, nurses, physiotherapist, speech therapist, patient management team (admin), etc.
+
+**Main user story:**
+
+- > As a physician, I want to see a summary of all relevant information at a glance, with an emphasis on the latest developments, so that I do not miss important facts. I want to be able to present a complete picture of the patient's current situation so that the medical team can agree on the next steps to ensure that the patient receives the best possible medical care.
```

Comment:
Provide more content, actually describe a ward visit and step by step what you are doing. What are you discussing, what are you viewing, what are you documenting.